packages/graph/hash_graph/lib/graph/src/ontology/types/uri.rs

```diff
@@ -23,13 +24,13 @@ impl fmt::Display for BaseUri {
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct VersionedUri {
     base_uri: BaseUri,
-    version: u32,
+    version: i32,
```

Comment:
We use `i32` in our database, it's easier to use it here as well. Happy to change if desired.

Reply:
To get rid of the casting everywhere I assume

Reply:
Yes 🙂

Reply:
We decided off-GitHub, that we want to use `u32` in public-facing API, otherwise it would be possible to create a negative version.